### ISSUE TITLE
nextdns windows app keeps crashing after pc boot

### ISSUE THREAD

**Issue by ijuliank** [nextdns/windows#39]
simple : the nextdns app freezes and doesnt respond

**Comment by rs** [comment]
Which version?

**Comment by ijuliank** [comment]
> Which version?

The main version when you download it from the my.nextdns.io

**Comment by rs** [comment]
Please try the beta: https://nextdns.io/download/beta

**Comment by ijuliank** [comment]
> Please try the beta: https://nextdns.io/download/beta

I’ll try again later this day :)

**Comment by ijuliank** [comment]
> Please try the beta: https://nextdns.io/download/beta

![image](https://user-images.githubusercontent.com/64658680/202920874-015863b7-ef25-4b1d-a480-8bc5a24fbe39.png)
cant access

**Comment by ijuliank** [comment]
@rs @Delaire

**Comment by rs** [comment]
https://nextdns.io/download/windows/beta

**Comment by ijuliank** [comment]
> https://nextdns.io/download/windows/beta

![image](https://user-images.githubusercontent.com/64658680/202921983-b44eff29-0161-490f-b322-69955ee48208.png)
same issue

**Comment by ijuliank** [comment]
2022/11/20 21:03:05 INFO starting NextDNSService service
2022/11/20 21:03:05 INFO Service starting
2022/11/20 21:03:05 INFO OnStart
2022/11/20 21:03:05 INFO Registry information Configuration: , UI: false, ARP: false
2022/11/20 21:03:05 INFO Service started
2022/11/20 21:03:05 INFO UI Connect: &{0xc00020a000 \\.\pipe\NextDNS}
2022/11/20 21:03:05 INFO send event: status map[locked:false state:stopped version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:05 INFO UI Connect: &{0xc000292000 \\.\pipe\NextDNS}
2022/11/20 21:03:05 INFO send event: status map[locked:false state:stopped version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:05 INFO received event: settings map[checkUpdates:true configuration:f58452 enabled:true error:<nil> locked:false reportDeviceName:true state:<nil> updateChannel:Stable upstreamHostName:<nil> version:<nil>]
2022/11/20 21:03:05 INFO UI Disconnect: &{0xc000292000 \\.\pipe\NextDNS}
2022/11/20 21:03:05 INFO Starting service
2022/11/20 21:03:05 INFO send event: status map[state:starting version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:06 INFO send event: status map[state:started version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:06 INFO Engine DNS engine started
2022/11/20 21:03:15 INFO UI Connect: &{0xc0000b6000 \\.\pipe\NextDNS}
2022/11/20 21:03:15 INFO send event: status map[locked:false state:started version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:15 INFO received event: settings map[checkUpdates:true configuration:f58452 enabled:true error:<nil> locked:false reportDeviceName:true state:<nil> updateChannel:Beta upstreamHostName:<nil> version:<nil>]
2022/11/20 21:03:15 INFO UI Disconnect: &{0xc0000b6000 \\.\pipe\NextDNS}
2022/11/20 21:03:15 INFO Starting service
2022/11/20 21:03:23 INFO UI Connect: &{0xc0002920b0 \\.\pipe\NextDNS}
2022/11/20 21:03:23 INFO send event: status map[locked:false state:started version:v3.0.5-0-gf10d5d7]
2022/11/20 21:03:23 INFO UI Disconnect: &{0xc0002920b0 \\.\pipe\NextDNS}
2022/11/20 21:03:23 INFO received event: settings map[checkUpdates:true configuration:f58452 enabled:true error:<nil> locked:false reportDeviceName:true state:<nil> updateChannel:Stable upstreamHostName:<nil> version:<nil>]
2022/11/20 21:03:23 INFO Starting service
2022/11/20 21:03:28 INFO Connected 45.90.28.0:443 (con=11ms tls=35ms, TCP, TLS13)
2022/11/20 21:03:28 INFO Connected 95.179.134.211:443 (con=10ms tls=15ms, TCP, TLS13)
2022/11/20 21:03:28 INFO Switching endpoint: https://windows.dns.nextdns.io#188.172.219.167,2a00:11c0:63:350::3,95.179.134.211,2001:19f0:5001:1faf:5400:2ff:fec8:7d49
2022/11/20 21:05:10 ERROR Engine DNS error message: 163:FWPS_PACKET_PREVIOUSLY_INJECTED_BY_SELF, code = 3221225473, param = 0
2022/11/20 21:06:44 INFO UI Connect: &{0xc0000b6370 \\.\pipe\NextDNS}
2022/11/20 21:06:44 INFO send event: status map[locked:false state:started version:v3.0.5-0-gf10d5d7]
2022/11/20 21:06:44 INFO received event: settings map[checkUpdates:true configuration:f58452 enabled:true error:<nil> locked:false reportDeviceName:true state:<nil> updateChannel:Stable upstreamHostName:<nil> version:<nil>]
2022/11/20 21:06:44 INFO UI Disconnect: &{0xc0000b6370 \\.\pipe\NextDNS}
2022/11/20 21:06:44 INFO Starting service

**Comment by ijuliank** [comment]
@rs

**Comment by ijuliank** [comment]
@Delaire i need a quick fix @rs

**Comment by rs** [comment]
The issue happens only when the UI is opened by double clicking the UI app. Going through the systray->settings work.

**Comment by ijuliank** [comment]
> The issue happens only when the UI is opened by double clicking the UI app. Going through the systray->settings work.

Thats not true, the crash happens every time when the pc is starting up and then it popups with unresponsive UI like shown in the screenshot

**Comment by rs** [comment]
Did you set a profile ID un the app?

**Comment by ijuliank** [comment]
> Did you set a profile ID un the app?

Yes

**Comment by ijuliank** [comment]
Seems like a fix isn’t going be there soon, and i’d like to have a refund
